LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Arnav Yash Chandra <arnavyash2004@outlook.com>
```

Comment:
This seems wrong but not sure what it should be.

Reply:
Yeah... we have
```
Copyright (c) 2021 Gnosis Service GmbH
```

In our other repo.